docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -110,8 +110,8 @@ ms.localizationpriority: medium
 
 | Method | Minimum<br>permission level | Details by mode | Minimum<br>requirement set |
 |---|---|---|:---:|
-| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-preview&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
-| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| addFileAttachmentAsync(uri, attachmentName, [options], [callback]) | **read/write item** | [Appointment Organizer](/javascript/api/outlook/office.appointmentcompose?view=outlook-js-preview&preserve-view=true#outlook-office-appointmentcompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
+| | | [Message Compose](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-addfileattachmentasync-member(1)) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md)<br>(classic Windows, Mac)<br><br>[1.8](../requirement-set-1.8/outlook-requirement-set-1.8.md)<br>(Web, new Outlook) |
```

Comment:
hmm, "new Outlook" doesn't make it clear this is on Windows - here and other instances

Reply:
Thanks for catching this. Yes, this should be "new Windows".